build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -295,3 +312,12 @@ String ddcModuleName(AssetId jsId) {
       : jsId.path;
   return jsPath.substring(0, jsPath.length - jsModuleExtension.length);
 }
+
+void fixupMetadata(Map<String, dynamic> json, String scratchPath, String path) {
```

Comment:
It doesn't look like the `path` arg is ever passed anything but `''` here, might as well remove it.

Also i would make this a private method.

Reply:
will do, thanks!